itou/templates/apply/table_for_siae.html

```diff
@@ -0,0 +1,98 @@
+                        {% endwith %}
+                    </h3>
+                    <div class="flex-column flex-md-row btn-group btn-group-sm btn-group-action" role="group" aria-label="Actions sur les candidatures">
+                        {% if filters_counter > 0 %}
+                            <a href="{% url 'apply:table_for_siae' %}" class="btn btn-secondary btn-ico mt-3 mt-md-0">
+                                <i class="ri-arrow-go-back-line" aria-hidden="true"></i>
+                                <span>Réinitialiser les filtres ({{ filters_counter }})</span>
+                            </a>
+                        {% endif %}
+                    </div>
+                </div>
+                <div class="table-responsive-lg mt-3 mt-md-4">
+                    <table class="table">
+                        <caption class="visually-hidden">Liste des informations candidats pour {{ company }}</caption>
+                        <thead>
+                            <tr>
+                                <th scope="col">Nom complet</th>
+                                <th scope="col">Niveau d'éducation</th>
+                                <th scope="col">RQTH</th>
+                                <th scope="col">RSA</th>
+                                <th scope="col">Editable Note</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            {% for job_application in job_applications_page %}
+                                <tr>
+                                    <td>
+                                        <a href="{% url 'apply:details_for_company' job_application_id=job_application.id %}?back_url={{ request.get_full_path|urlencode }}"
+                                           aria-label="Voir la candidature de {{ job_application.job_seeker.get_full_name|mask_unless:job_application.user_can_view_personal_information }}"
+                                           {% matomo_event "emplois-premium" "clic" "voir-candidature-employeur" %}>
+                                            {{ job_application.job_seeker.get_full_name|mask_unless:job_application.user_can_view_personal_information }}
+                                        </a>
+                                    </td>
+                                    <td>{{ job_application.job_seeker.jobseeker_profile.get_education_level_display }}</td>
+                                    <td>{{ job_application.job_seeker.jobseeker_profile.rqth_employee|yesno:"Oui,Non" }}</td>
+                                    <td>{{ job_application.job_seeker.jobseeker_profile.get_has_rsa_allocation_display }}</td>
+                                    <td>
+                                        {# vincentporte - mars 2024 - assumed not using `only` params when calling this include, because of csrf_token injection #}
```

Comment:
point pour revue : `csrf_token` versus `only` dans l'appel au template

Reply:
Pourquoi pas :
```diff
diff --git a/itou/templates/apply/table_for_siae.html b/itou/templates/apply/table_for_siae.html
index eb6bd69c7..b253d50ba 100644
--- a/itou/templates/apply/table_for_siae.html
+++ b/itou/templates/apply/table_for_siae.html
@@ -76,8 +76,7 @@
                                     <td>{{ job_application.job_seeker.jobseeker_profile.rqth_employee|yesno:"Oui,Non" }}</td>
                                     <td>{{ job_application.job_seeker.jobseeker_profile.get_has_rsa_allocation_display }}</td>
                                     <td>
-                                        {# vincentporte - mars 2024 - assumed not using `only` params when calling this include, because of csrf_token injection #}
-                                        {% include 'premium/partials/save_note_form.html' with job_application_id=job_application.id content=job_application.premium_note.content %}
+                                        {% include 'premium/partials/save_note_form.html' with job_application_id=job_application.id content=job_application.premium_note.content csrf_token=csrf_token only %}
                                     </td>
                                 </tr>
                             {% endfor %}
```